Formula/python@3.8.rb

```diff
@@ -0,0 +1,287 @@
+class PythonAT38 < Formula
+  desc "Interpreted, interactive, object-oriented programming language"
+  homepage "https://www.python.org/"
+  url "https://www.python.org/ftp/python/3.8.0/Python-3.8.0.tar.xz"
+  sha256 "b356244e13fb5491da890b35b13b2118c3122977c2cd825e3eb6e7d462030d84"
+  # head "https://github.com/python/cpython.git" # Commented out for migration period (3.7 -> 3.8)
```

Comment:
Please just remove comment and the head link. I think it's better to not have too much commented out code. We can re-add whatever is needed later.